src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -81,12 +82,21 @@ const ArticlePageMostReadSection = styled(MostReadSection)`
 
 const ArticlePage = ({ pageData, mostReadEndpointOverride }) => {
   const { articleAuthor } = useContext(ServiceContext);
+  const { state } = useLocation();
+  const anchor = state?.anchor;
   const headline = getHeadline(pageData);
   const description = getSummary(pageData) || getHeadline(pageData);
   const firstPublished = getFirstPublished(pageData);
   const lastPublished = getLastPublished(pageData);
   const aboutTags = getAboutTags(pageData);
 
+  useEffect(() => {
+    if (anchor) {
+      document.getElementById(anchor)?.scrollIntoView();
```

Comment:
instead of querying the DOM twice you could make `document.getElementById(anchor)` a constant

Reply:
Thanks for the review @jroebu14